packages/adaptive_scaffold/example/lib/adaptive_layout_demo.dart

```diff
@@ -54,54 +54,200 @@ class MyHomePage extends StatelessWidget {
           icon: Icon(Icons.video_call_outlined, color: Colors.black)),
     ];
 
-    // AdaptiveLayout has a number of slots that take SlotLayouts and these
-    // SlotLayouts' configs take maps of Breakpoints to SlotLayoutConfigs.
+    Widget trailingNavRail = Column(
+      children: [
+        const Divider(
+          color: Colors.black,
+        ),
+        const SizedBox(
+          height: 10,
+        ),
+        Row(
+          children: const [
+            SizedBox(
+              width: 27,
+            ),
+            Text(
+              "Folders",
+              style: TextStyle(fontSize: 16),
+            ),
+          ],
+        ),
+        const SizedBox(
+          height: 10,
+        ),
+        Row(
+          children: [
+            const SizedBox(
+              width: 16,
+            ),
+            IconButton(
+              onPressed: () {},
+              icon: const Icon(Icons.folder_copy_outlined),
+              iconSize: 21,
+            ),
+            const SizedBox(
+              width: 21,
+            ),
+            const Text("Freelance"),
+          ],
+        ),
+        const SizedBox(
+          height: 12,
+        ),
```

Comment:
The type of formatting is repeated pretty often. It might make more sense to make a widget that takes the properties that change and use that instead.